testing/perl-crypt-openssl-aes/APKBUILD

```
@@ -0,0 +1,39 @@
+# Automatically generated by apkbuild-cpan, template 1
+# Maintainer: Sergey Safarov <s.safarov@gmail.com>
+pkgname=perl-crypt-openssl-aes
+_realname=Crypt-OpenSSL-AES
+pkgver=0.02
+pkgrel=0
+pkgdesc="Perl module Crypt-OpenSSL-AES"
+url="http://search.cpan.org/~ttar/Crypt-OpenSSL-AES-0.02/"
+arch="all"
+license="Artistic-1.0 GPL+"
+cpandepends=""
+cpanmakedepends=""
+depends="$cpandepends"
+makedepends="perl-dev openssl-dev $cpanmakedepends"
+install=
+subpackages="$pkgname-doc"
+source="http://search.cpan.org/CPAN/authors/id/T/TT/TTAR/$_realname-$pkgver.tar.gz"
+
+_builddir="$srcdir"/$_realname-$pkgver
+
+build() {
+	cd "$_builddir"
+	export CFLAGS=`perl -MConfig -E 'say $Config{ccflags}'`
```

Its a very minor point but many of the Alpine Linux developers prefer to use '$()' over '``'
```suggestion
	export CFLAGS=$(perl -MConfig -E 'say $Config{ccflags}')
```